databases/backends/postgres.py

```diff
@@ -104,8 +104,9 @@ def __init__(
         self._dialect = dialect
         self._column_map, self._column_map_int, self._column_map_full = column_maps
 
-    def values(self) -> typing.ValuesView:
-        return self._row.values()
+    @property
+    def _mapping(self) -> asyncpg.Record:
+        return self._row
```

Comment:
I would like your insight on this matter.

For `postgres`, `row.values()` was still working thanks to this `Record` class but for all other backends we now need to do `row._mapping.values()` with sqlalchemy >= 1.4

Do we want to:
1) keep `values()` only for `postgres`
2) keep `values()` for all backends and we can attach the deprecated `property` to the `sqlalchemy.Row` class
3) let it like this and mimic sqlalchemy API also for `postgres`

Thanks for this very nice lib!

Reply:
imo It'd be better to keep `row.values()` for all backends since looking at test changes I can see that `row.keys()` is still present so `row._mapping.values()` would bring an inconsistency

Reply:
Hmmm I don't really want to add `row.values()` for all backends since it would need some monkeypatching or wrapper around the SQLAlchemy objects.
I reckon the idea of `databases` is to keep things simple and to stay close to SQLAlchemy when possible.
This is why I went with updating postgres backend only to mimic SQLAlchemy behaviour

Reply:
My bad, I started my comment with keep since that inconsistency was triggering me, although after reading SQLAlchemy's changelogs and docs I immediately changed my mind about keeping `values` but did not change the original paragraph. 

I do agree with decision to keep the API as close to SQLAlchemy's as possible.

Just that test having both `row.values` and `row._mapping.keys` misled me.